src/components/odata/analyze.vue

```diff
@@ -10,7 +10,7 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <modal id="odata-analyze" :state="state" hideable backdrop
+  <modal v-if="!config.oidcEnabled" id="odata-analyze" :state="state" hideable backdrop
```

Comment:
This `v-if` is a nice-to-have. With the change to `OdataDataAccess`, the `state` prop of this component will never be toggled to `true`, so this modal will never be shown. What this `v-if` does is also remove this unnecessary content from the DOM if SSO is enabled. We've done something similar in another case for SSO, with the `UserResetPassword` modal.

Relatedly, I didn't add a test for this change: I test the change to `OdataDataAccess`, but not that the content here isn't rendered. I don't think such a test is really needed, because this change is more of an optimization and isn't user-facing. What's most important to test is that the button and link aren't shown.